DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Exception/DicomAnonymizationException.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Exceptions
+{
+    public class DicomAnonymizationException : Exception
+    {
+        public DicomAnonymizationException(DicomAnonymizationErrorCode templateManagementErrorCode, string message)
+            : base(message)
+        {
+            TemplateManagementErrorCode = templateManagementErrorCode;
+        }
+
+        public DicomAnonymizationException(DicomAnonymizationErrorCode templateManagementErrorCode, string message, Exception innerException)
+            : base(message, innerException)
+        {
+            TemplateManagementErrorCode = templateManagementErrorCode;
+        }
+
+        public DicomAnonymizationErrorCode TemplateManagementErrorCode { get; }
```

Comment:
Is this related with template management (for anonymizer)?

Reply:
It is a big typo